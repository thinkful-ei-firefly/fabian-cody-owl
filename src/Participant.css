.participant-list{
  /* display: none; 
  display: flex;*/
  flex-direction: column;
}

.participant{
  text-align: center;
}

.participant p{
  margin: 3px;
}

.participant-list img{
  border-radius: 50%; 
  width:60px; 
  height:60px; 
}