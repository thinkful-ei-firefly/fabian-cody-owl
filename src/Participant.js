import React from 'react';

class Participant extends React.Component{


	render(){
		return (
			<div className={this.props.inSession?'session-participant':''}>
			<img src={this.props.avatar} alt='avatar'/>
			<p>{this.props.name}</p>
			<span>{this.props.onStage?'on stage':'off stage'}</span>
			</div>
			)
	};
}

export default Participant;