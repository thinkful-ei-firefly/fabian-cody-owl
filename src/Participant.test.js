import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Participant from './Participant';

describe('<Participant />', () => {
	test('Renders a Participant', () => {
		const div = document.createElement('div');
		ReactDOM.render(<Participant key="1" avatar="path" name="Mark" onStage={true}/>, div);
		ReactDOM.unmountComponentAtNode(div);
	})

	test('Render Participant Snap', () => {
		const tree = renderer.create(<Participant key="1" avatar="path" name="Mark" onStage={true}/>).toJSON();
		expect(tree).toMatchSnapshot();
	})
})
