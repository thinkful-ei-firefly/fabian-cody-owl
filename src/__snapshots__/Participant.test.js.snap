// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Participant /> Render Participant Snap 1`] = `
<div
  className="participant"
>
  <img
    alt="avatar"
    src="path"
  />
  <p>
    Mark
  </p>
  <p>
    on stage
  </p>
</div>
`;
